modular_skyrat/modules/mapping/code/mob_spawns.dm

```diff
@@ -501,10 +501,10 @@
 //CRYO CONSOLES
 /obj/machinery/computer/cryopod/interdyne
 	radio = /obj/item/radio/headset/interdyne
-	announcement_channel = RADIO_CHANNEL_INTERDYNE
+	announcement_channel = "Interdyne"
```

Comment:
Why can't those be defines? If the content of the define wasn't correct, just make a new one with the proper name in it.

Reply:
Announcement channel was set to null upstream, and this is the only reason why I can think of, and it works on my machine:tm:, so yea...

Reply:
on live it is now not null, instead it's not announcing it at all